nntrainer/graph/network_graph.cpp

```diff
@@ -736,6 +747,25 @@ void NetworkGraph::inPlaceOptimize() {
   }
 }
 
+/**
+ * @brief Set the Inplace Shared Memory Config By Layer object
+ *
+ * @param lnode layer node object
+ * @param shared_var if the variable should be shared
+ * @param shared_grad if the gradient should be shared
+ */
+static void
+setInplaceSharedMemoryConfigByLayer(const std::shared_ptr<LayerNode> &lnode,
+                                    bool &shared_var, bool &shared_grad) {
+  if (lnode->getType() == MultiOutLayer::type) {
```

Comment:
what else layers are in this kind? split? 
and the concat, addition layer can be in shared_var and shared_grad=true? 
Activation can be share every thing. 
Could you listed up in comment?

Reply:
I will add as a comment.